python/pipeline/experiment.py

```diff
@@ -374,11 +336,31 @@ class SurgeryOutcome(dj.Lookup):
         ['Non-Survival'],
     ]
 
+@schema
+class Mice(dj.Manual):
```

Comment:
Generally we keep the name of the table singular to refer to what one entry is about.

Reply:
Also I'm imagining that having own Mice table would now make it unnecessary to have `commons`? Where is the `path` table being used?

Reply:
thanks, i will change `mice` to `mouse`.

agreed, if we bring over `mice`, `path`, and `virus`, then we can remove the dependency on `commons`.  `path` is being used in `experiment.py`, `odor.py`, `posture.py`, `pupil.py`, `temperature.py`, `treadmill.py`, and `vreso.py`.

Reply:
awesome, sounds good